policy/modules/system/userdomain.if

```diff
@@ -3612,6 +3612,44 @@ interface(`userdom_delete_all_user_runtime_named_sockets',`
 	allow $1 user_runtime_content_type:sock_file delete_sock_file_perms;
 ')
 
+########################################
+## <summary>
+##	delete user runtime blk files
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`userdom_delete_all_user_runtime_blk_files',`
+	gen_require(`
+		attribute user_runtime_content_type;
+	')
+
+	allow $1 user_runtime_content_type:dir list_dir_perms;
+	allow $1 user_runtime_content_type:blk_file delete_blk_file_perms;
+')
+
+########################################
+## <summary>
+##	delete user runtime chr files
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`userdom_delete_all_user_runtime_chr_files',`
+	gen_require(`
+		attribute user_runtime_content_type;
+	')
+
+	allow $1 user_runtime_content_type:dir list_dir_perms;
+	allow $1 user_runtime_content_type:chr_file delete_chr_file_perms;
+')
```

Comment:
same here: why not use `delete_chr_files_pattern()`?

Reply:
Same as above.